errors.js

```diff
@@ -17,9 +17,15 @@ var BackendsRequired = TypedError({
         'on opts.\n' +
         'Ensure you call `Logger({ backends: ... })`.\n'
 });
+var LevelRequired = TypedError({
+    type: 'rt-logger.child-logger-requires-additional-level',
```

Comment:
suggestion: `logtron.x.y`

Old type fields cannot be changed because of back compat. However we should follow best practices for new type fields :)